README.md

```diff
@@ -1,20 +1,37 @@
+[![CharmHub Badge](https://charmhub.io/jenkins-agent/badge.svg)](https://charmhub.io/jenkins-agent)
+[![Publish to edge](https://github.com/canonical/jenkins-agent-operator/actions/workflows/publish_charm.yaml/badge.svg)](https://github.com/canonical/jenkins-agent-operator/actions/workflows/publish_charm.yaml)
+[![Promote charm](https://github.com/canonical/jenkins-agent-operator/actions/workflows/promote_charm.yaml/badge.svg)](https://github.com/canonical/jenkins-agent-operator/actions/workflows/promote_charm.yaml)
+
+A [Juju](https://juju.is/) [charm](https://juju.is/docs/olm/charmed-operators) deploying and managing Jenkins agents on machines. It is configurable to integrate with a Jenkins charm deployed in another Juju model.
 
 This charm simplifies initial deployment and "day N" operations of Jenkins Agent on VMs and bare metal.
 
 As such, the charm makes it easy for those looking to take control of their own agents whilst keeping operations simple, and gives them the freedom to deploy on the platform of their choice.
 
 For DevOps or SRE teams this charm will make operating Jenkins Agent simple and straightforward through Juju's clean interface. It will allow easy deployment into multiple environments for testing changes, and supports scaling out for enterprise deployments.
 
+For information about how to deploy, integrate, and manage this charm, see the Official [jenkins-agent charm Documentation](https://charmhub.io/jenkins-agent/docs).
+
+## Get started
+
+To begin, refer to the [Getting Started](https://charmhub.io/jenkins-agent/docs/tutorial-getting-started) tutorial for step-by-step instructions.
+
+### Basic operations
+
+#### Integrate with the Jenkins-k8S charm
+
+See the [deploy and integrate machine agents](https://charmhub.io/jenkins-k8s/docs/tutorial-getting-started#deploy-and-integrate-machine-agents-optional) section in the jenkins-k8s-operator documentation.
+
+## Learn more
+
+- [Read more](https://charmhub.io/jenkins-agent/docs)
+
 ## Project and community
 
 The Jenkins-agent Operator is a member of the Ubuntu family. It's an open source project that warmly welcomes community projects, contributions, suggestions, fixes and constructive feedback.
 
-- [Code of conduct](https://ubuntu.com/community/code-of-conduct)
-- [Get support](https://discourse.charmhub.io/)
-- [Join our online chat](https://app.element.io/#/room/#charmhub-charmdev:ubuntu.com)
-- [Contribute](Contribute)
-
-Thinking about using the Jenkins-k8s Operator for your next project? [Get in touch](https://app.element.io/#/room/#charmhub-charmdev:ubuntu.com)!
+* [Issues](https://github.com/canonical/jenkins-agent-operator/issues) <!--Link to GitHub issues (if applicable)-->
+* [Contributing](https://charmhub.io/jenkins-agent#contributing-to-this-documentation) <!--Link to any contribution guides--> 
```

Comment:
Could you link https://github.com/canonical/jenkins-agent-operator/blob/main/CONTRIBUTING.md instead?

Reply:
Updated the link